VeroliBGV/BGV009.xml

```diff
@@ -0,0 +1,1084 @@
+                                    <note>The quire is a bifolium inserted at the end of <ref target="#q9"/>.
+                                    </note>
+                                 </item>
+                              </list>
+                           </collation>
+                        </supportDesc>
+                        
+                        <layoutDesc>
+                           <layout columns="2" writtenLines="18 21">
+                              <locus from="76ra" to="77ra"/>
+                              <note corresp="#textarea #margin">Data on text area and margin dimensions are taken from <locus target="#76r"/>.</note>
+                              <dimensions unit="mm" xml:id="textarea2">
+                                 <height>141</height>
+                                 <width>141</width>  
+                              </dimensions>
+                              <dimensions type="margin" unit="mm" xml:id="margin2">
+                                 <dim type="top">18</dim>
+                                 <dim type="bottom">81</dim>
+                                 <dim type="right">26</dim>
+                                 <dim type="left">8</dim>
+                                 <dim type="intercolumn">13</dim> 
+                              </dimensions>
+                              <note> The characters per line are 9-11.</note>
+                              <ab type="pricking">Ruling and pricking are barely visible. Primary pricks are visible; ruling pricks are not visible.</ab>
+                              <ab type="ruling" subtype="pattern">Ruling pattern: 1A-1A-1A1A/0-0/0-0/C.</ab>
+                              <ab type="ruling">The upper line is written above the ruling. The bottom line is written above the ruling.</ab>
+                           </layout>
+                        </layoutDesc>
+                     </objectDesc>
+                   <additions>
+                        <list>                   
+                           <item xml:id="e4">
+                              <desc>A few corrections placed interlineally (e.g. on <locus target="#76ra #76rb #76va"/>).</desc>
+                           </item>
+                        </list>
+                     </additions>
+                  </physDesc>
+                  <history>
+                     <origin>
+                        <origDate notBefore="1700" notAfter="1900" evidence="lettering">
+                           The handwriting is consistent with a 18th/19th-century dating.
+                        </origDate>
+                     </origin>                   
+                  </history>
+               </msPart>
+               
+          
+             <msPart xml:id="p3">
+                  <msIdentifier>
+                  </msIdentifier>
+                  <msContents>
+                     <summary/>
+                     <msItem xml:id="p3_i1">
```

Comment:
Here again, I would first give the general textual unit 
```
<msItem xml:id="p3_i1">
<locus from="78ra" to="87vb"/>
<title type="complete" ref="LIT4315Miracles"/>
```

And then the two miracles as sections `<msItem xml:id="p3_i1.1 ">` and `<msItem xml:id="p3_i1.2">`